### PR TITLE
Parquet parser return Decimal fields as strings

### PR DESCRIPTION
**Pull request by girarda**
## What
* Update the parquet parser so it returns Decimal values as strings by default 

## How
* Call ParquetParser._to_output_value on each field before yielding the record

## Recommended reading order
1. `airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/parquet_parser.py`
2. `airbyte-cdk/python/unit_tests/sources/file_based/scenarios/parquet_scenarios.py`
3. `airbyte-cdk/python/unit_tests/sources/file_based/test_scenarios.py`
4. `airbyte-cdk/python/unit_tests/sources/file_based/scenarios/avro_scenarios.py`

### CODE REVIEW

File: airbyte-cdk/python/unit_tests/sources/file_based/scenarios/avro_scenarios.py [hunk 1]
```diff
@@ -405,8 +405,6 @@
                     "col_time_micros": "12:00:00.456789",
                     "col_timestamp_millis": "2022-05-29T00:00:00.456000+00:00",
                     "col_timestamp_micros": "2022-05-30T00:00:00.456789+00:00",
-                    "col_local_timestamp_millis": "2022-05-29T00:00:00.456000",
```
Comment (girarda) [inline airbyte-cdk/python/unit_tests/sources/file_based/scenarios/avro_scenarios.py:408]: the file doesn't have these fields

File: airbyte-cdk/python/unit_tests/sources/file_based/test_scenarios.py [hunk 2]
```diff
@@ -269,9 +271,10 @@ def _verify_read_output(output: Dict[str, Any], scenario: TestScenario) -> None:
     assert len(records) == len(expected_records)
     for actual, expected in zip(records, expected_records):
         if "record" in actual:
+            assert len(actual["record"]["data"]) == len(expected["data"])
```
Comment (girarda) [inline airbyte-cdk/python/unit_tests/sources/file_based/test_scenarios.py:274]: confirm the record has all the expected fields

File: airbyte-cdk/python/unit_tests/sources/file_based/test_scenarios.py [hunk 3]
```diff
@@ -269,9 +271,10 @@ def _verify_read_output(output: Dict[str, Any], scenario: TestScenario) -> None:
...
             for key, value in actual["record"]["data"].items():
                 if isinstance(value, float):
-                    assert math.isclose(value, float(expected["data"][key]), abs_tol=1e-04)
+                    assert math.isclose(value, expected["data"][key], abs_tol=1e-04)
```
Comment (girarda) [inline airbyte-cdk/python/unit_tests/sources/file_based/test_scenarios.py:277]: do not cast. the test should fail if the type doesn't match

File: airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/parquet_parser.py [hunk 4]
```diff
@@ -55,11 +55,15 @@ def parse_records(
...
-                batch_dict = reader.read_row_group(row_group).to_pydict()
-                for record_values in zip(*batch_dict.values()):
-                    record = dict(zip(batch_dict.keys(), record_values))
-                    record.update(partition_columns)
-                    yield record
+                batch = reader.read_row_group(row_group)
+                for row in range(batch.num_rows):
+                    yield {
+                        **{
+                            column: ParquetParser._to_output_value(batch.column(column)[row], parquet_format)
```
Comment (girarda) [inline airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/parquet_parser.py:62]: get the value from the batch so we still have its type

**Review by brianjlai — APPROVED** [review]

**Review by clnoll — APPROVED** [review]
LGTM @girarda!

**Pull request merged**